torch/csrc/jit/profiling_record.cpp

```diff
@@ -18,7 +18,67 @@ ProfileOp* ProfilingRecord::createProfileNode(
   return pn;
 }
 
-void ProfilingRecord::instrumentBlock(Block* block) {
+static void unprofileGraphInputs(const std::shared_ptr<Graph> &graph) {
+  for (auto i : graph->inputs()) {
+    if (i->type()->isSubtypeOf(TensorType::get())) {
+      i->setType(TensorType::get());
+    }
+  }
+}
+
+static void unprofileBlock(Block *block) {
+  for (auto it = block->nodes().begin(); it != block->nodes().end(); ++it) {
+    auto n = *it;
+    for (auto o : n->outputs()) {
+      if (o->type()->isSubtypeOf(TensorType::get())) {
```

Comment:
Probably want to use unshapedType, what happens if Tensors are an element of a tuple or list?